src/main/java/org/team1540/robot2023/commands/arm/Arm.java

```diff
@@ -76,5 +86,23 @@ public void setExtension(double extension) {
     public void stopAll() {
         pivot1.set(ControlMode.PercentOutput, 0);
         telescope.set(ControlMode.PercentOutput, 0);
+        extending = false;
+    }
+
+    private void limitArmExtension(){
+        if(extending){
```

Comment:
This would not re-legalize arm position if it was motionless but still in an illegal state, we probably want to account for this